wiki/about/moderators.md

```diff
@@ -92,3 +92,22 @@ Any stroopwafels?
 **Roles:** Moderator  
 
 [Twitch](https://twitch.tv/olafstad)
+
+## Chara_
+gaming 101
+
+**Roles:** Support, 3D Artists, Moderator
+
+[Discord](https://discord.gg/NXnPYEh) | [Youtube](https://www.youtube.com/c/CharaHere)
```

Comment:
Are links to discords allowed?

Reply:
I mean, the discord is relevant to beat saber

Reply:
I don't see why not

Reply:
I was just wondering. Thanks!